### PR TITLE
Prepare final release 1.0.0

### PR DESCRIPTION
**Pull request by notoraptor**
@lamblin @nouiz

### CODE REVIEW

File: NEWS.txt [hunk 1]
```diff
@@ -3,26 +3,174 @@ Release Notes
...
+new features, interface changes, improvements and bug fixes.
 
-We recommend that every developer updates to this version.
+We recommend that everybody update to this version.
 
-Highlights:
+Highlights (since 0.9.0):
+ - Announcing that `MILA will stop developing Theano <https://groups.google.com/d/msg/theano-users/7Poq8BZutbY/rNCIfvAEAwAJ>`_
+ - conda packages now available and updated in our own conda channel ``mila-udem``
+   To install it: ``conda install -c mila-udem -c mila-udem/label/pre theano pygpu``
```
Comment (abergeron) [inline NEWS.txt]: This shouldn't specify mila-udem/label/pre for the final release

File: NEWS.txt [hunk 2]
```diff
@@ -3,26 +3,174 @@ Release Notes
...
+ - Support NumPy ``1.13``
+ - Support pygpu ``0.7``
+ - Moved Python ``3.*`` minimum supported version from ``3.3`` to ``3.4``
+ - Added conda recipe
+ - Replaced deprecated package ``nose-parameterized`` with up-to-date package ``parameterized`` for Theano requirements
+ - Theano now internally uses ``sha256`` instead of ``md5`` to work on systems that forbide ``md5`` for security reason
```
Comment (abergeron) [inline NEWS.txt]: forbide -> forbid

File: NEWS.txt (hunk 1 above)
Comment (notoraptor) [reply NEWS.txt]: For theano, when I use only `-c mila-udem`, conda suggests theano `0.9`:
```
(py3) C:\Users\notoraptor\mila\dev\git\theano>conda install -c mila-udem theano
Fetching package metadata .................
Solving package specifications: .

Package plan for installation in environment C:\Users\notoraptor\Anaconda3\envs\py3:

The following NEW packages will be INSTALLED:

    icc_rt:                         2017.0.4-h97af966_0
    intel-openmp:                   2018.0.0-hcd89f80_7
    libgpuarray:                    0.7.5-vc14h07fe1f2_0  mila-udem [vc14]
    libpython:                      2.0-py36_0
    m2w64-binutils:                 2.25.1-5
    m2w64-bzip2:                    1.0.6-6
    m2w64-crt-git:                  5.0.0.4636.2595836-2
    m2w64-gcc:                      5.3.0-6
    m2w64-gcc-ada:                  5.3.0-6
    m2w64-gcc-fortran:              5.3.0-6
    m2w64-gcc-libgfortran:          5.3.0-6
    m2w64-gcc-libs:                 5.3.0-7
    m2w64-gcc-libs-core:            5.3.0-7
    m2w64-gcc-objc:                 5.3.0-6
    m2w64-gmp:                      6.1.0-2
    m2w64-headers-git:              5.0.0.4636.c0ad18a-2
    m2w64-isl:                      0.16.1-2
    m2w64-libiconv:                 1.14-6
    m2w64-libmangle-git:            5.0.0.4509.2e5a9a2-2
    m2w64-libwinpthread-git:        5.0.0.4634.697f757-2
    m2w64-make:                     4.1.2351.a80a8b8-2
    m2w64-mpc:                      1.0.3-3
    m2w64-mpfr:                     3.1.4-4
    m2w64-pkg-config:               0.29.1-2
    m2w64-toolchain:                5.3.0-7
    m2w64-tools-git:                5.0.0.4592.90b8472-2
    m2w64-windows-default-manifest: 6.4-3
    m2w64-winpthreads-git:          5.0.0.4634.697f757-2
    m2w64-zlib:                     1.2.8-10
    mako:                           1.0.7-py36he15cdb7_0
    markupsafe:                     1.0-py36h0e26971_1
    mkl:                            2018.0.0-h36b65af_4
    mkl-service:                    1.1.2-py36h57e144c_4
    msys2-conda-epoch:              20160418-1
    numpy:                          1.13.3-py36ha320f96_0
    pygpu:                          0.7.5-py36he104453_0  mila-udem
    scipy:                          1.0.0-py36h1260518_0
    six:                            1.11.0-py36h4db2310_1
    theano:                         0.9.0-py36_0

Proceed ([y]/n)? n

Exiting
```

With `-c mila-udem/label/pre`, it suggests the expected version:
```
(py3) C:\Users\notoraptor\mila\dev\git\theano>conda install -c mila-udem/label/pre theano
Fetching package metadata .................
Solving package specifications: .

Package plan for installation in environment C:\Users\notoraptor\Anaconda3\envs\py3:

The following NEW packages will be INSTALLED:

    icc_rt:                         2017.0.4-h97af966_0
    intel-openmp:                   2018.0.0-hcd89f80_7
    libgpuarray:                    0.7.2.dev0-vc14he8c4eb7_0 mila-udem/label/pre [vc14]
    libpython:                      2.0-py36_0
    m2w64-binutils:                 2.25.1-5
    m2w64-bzip2:                    1.0.6-6
    m2w64-crt-git:                  5.0.0.4636.2595836-2
    m2w64-gcc:                      5.3.0-6
    m2w64-gcc-ada:                  5.3.0-6
    m2w64-gcc-fortran:              5.3.0-6
    m2w64-gcc-libgfortran:          5.3.0-6
    m2w64-gcc-libs:                 5.3.0-7
    m2w64-gcc-libs-core:            5.3.0-7
    m2w64-gcc-objc:                 5.3.0-6
    m2w64-gmp:                      6.1.0-2
    m2w64-headers-git:              5.0.0.4636.c0ad18a-2
    m2w64-isl:                      0.16.1-2
    m2w64-libiconv:                 1.14-6
    m2w64-libmangle-git:            5.0.0.4509.2e5a9a2-2
    m2w64-libwinpthread-git:        5.0.0.4634.697f757-2
    m2w64-make:                     4.1.2351.a80a8b8-2
    m2w64-mpc:                      1.0.3-3
    m2w64-mpfr:                     3.1.4-4
    m2w64-pkg-config:               0.29.1-2
    m2w64-toolchain:                5.3.0-7
    m2w64-tools-git:                5.0.0.4592.90b8472-2
    m2w64-windows-default-manifest: 6.4-3
    m2w64-winpthreads-git:          5.0.0.4634.697f757-2
    m2w64-zlib:                     1.2.8-10
    mako:                           1.0.7-py36he15cdb7_0
    markupsafe:                     1.0-py36h0e26971_1
    mkl:                            2018.0.0-h36b65af_4
    mkl-service:                    1.1.2-py36h57e144c_4
    msys2-conda-epoch:              20160418-1
    numpy:                          1.13.3-py36ha320f96_0
    pygpu:                          0.7.2.dev0-py36hcd4186d_0 mila-udem/label/pre
    scipy:                          1.0.0-py36h1260518_0
    six:                            1.11.0-py36h4db2310_1
    theano:                         1.0.0rc1-py36hf6ce46d_0   mila-udem/label/pre

Proceed ([y]/n)? n

Exiting
```

Comment (abergeron) [reply NEWS.txt]: That's because the final release isn't out yet.  When it will be it will work.

Comment (notoraptor) [reply NEWS.txt]: Ok !

**Comment by notoraptor** [comment]
@abergeron fixed.

**Comment by lamblin** [comment]
Btw, should we require pygpu >= 0.7.5 in the conda package?

**Comment by abergeron** [comment]
>= 0.7.0 is ok.  Some interesting fixes were made, but none are 100% required as far as I know.

Anyway conda chooses the most recent version when installing a package so this shouldn't be a huge hurdle.

**Comment by lamblin** [comment]
OK.

**Pull request merged**